src/app/resources/learning-resources/learning-resources.component.ts

```diff
@@ -9,7 +9,7 @@ export class LearningResourcesComponent implements OnInit {
 
   constructor() { }
 
-  ngOnInit() {
+  ngOnInit() : void {
```

Comment:
Please remove the extra space before the colon as this introduces a new lint error. You can run `npm run lint` and verify no lint errors display for the files listed in this issue. Thanks!